spec/Section 4 -- Introspection.md

```diff
@@ -141,6 +141,9 @@ type __Type {
   # should be non-null for ENUM only, must be null for the others
   enumValues(includeDeprecated: Boolean = false): [__EnumValue!]
 
+  # should be non-null for INPUT_OBJECT only, must be null for the others
+  requiresExactlyOneField: Boolean!
```

Comment:
Should not have a `!` here.

Reply:
Sorry for asking here. 
Thanks for the features. But, do you know why @oneOf calls as unknown directive in my project, the same behavior with inputunion. Maybe I'm doing somethiing wrong
@benjie

Reply:
GraphQL does not have @oneOf or inputUnion yet; we’re actively working on picking a solution to add to the GraphQL specification. If you’re interested in helping out, please contribute to the input union RFC document.

Reply:
Thanks for the answer. I'm actually interested in to help you guys, but actually that's my first experience of kinda contribution. That is why I need mentor or someone to take tasks or telling me what to do)

For clarification, for the current version we can only use union to read data, make some queries and not to write. And there is no working solution to realize it now. Right?

Reply:
Yes; there's polymorphic output types but no polymorphic input types. You can read more in the problem statement: https://github.com/graphql/graphql-spec/blob/master/rfcs/InputUnion.md#-problem-statement Have a look through that document and see if you think there's anything you can add that would help move the conversation forward.